sdk/nodejs/runtime/resource.ts

```diff
@@ -19,8 +19,10 @@ import * as utils from "../utils";
 
 import { getAllResources, Input, Inputs, Output, output } from "../output";
 import { ResolvedResource } from "../queryable";
+import { expandProviders } from "../resource";
```

Comment:
Hopefully this doesn't result in circular import problems, since `resource.ts` imports some things from `runtime/resource.ts`. 🤞

Reply:
All the tests passed, so seems like it should be ok.